src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.Android.cs

```diff
@@ -54,7 +54,7 @@ private void OnMenuItemClick(object sender, PopupMenu.MenuItemClickEventArgs e)
 			var items = Items.OfType<MenuFlyoutItem>().Where(i => i.Visibility == Visibility.Visible).ToArray();
 			var item = items[e.Item.ItemId];
 
-			item.Command.ExecuteIfPossible(item.CommandParameter);
+			item.InvokeClick(); 
```

Comment:
I know it's not something you changed, but I'm curious why this pattern is only used for Android...

Reply:
It is mystery for me also. But change I made - makes Click working :)
Seems like shared MenuFlyoutItem has prepared method InvokeClick, but .Android. doesn't use this method, and tries only to Command.Execute (without Click).
And as it doesn't change shared file in any meaningful way, it should work for others also. If MenuFlyoutItem works for other platforms :)

Reply:
Should this conversation be marked as 'resolved'?

Reply:
I just need to know you tested it on both iOS and Wasm... ;-)

Reply:
I didn't test it 'on my side', it should be tested by CI :)

Reply:
The CI can only test what we code it to test. When we make a change, we need to manually certify it works.

If you don't have a mac connected, you should at least test on Wasm, which is easy to test on your side.

Reply:
CI fails, but not on my test, and I don't know how to check which test is failing (and I don't know if this test failure is related to this small change)

Reply:
Maybe test fails because of issue #2035...